packages/cdc-react/src/components/Dropdown/Dropdown.tsx

```diff
@@ -30,13 +30,18 @@ export const Dropdown = ({
   onKeyDownDropdownLabel,
   onKeyDownDropdownItems,
   id,
-}: DropdownProps) => {
+  className,
+}: DropdownProps & JSX.IntrinsicElements["div"]) => {
   const [dropdownOpen, setDropdownVisibility] = useState(false);
   const [dropdownCurrentItem, setDropdownCurrentItem] = useState(label);
 
+  const classList = ["dropdown"];
```

Comment:
I think we can simplify this to just an interpolated string. Since there is only one conditional class. We can delete this piece and just the following in the div below. 
```
<div className={`dropdown ${className}`}>
```